queries.sql

```diff
@@ -82,3 +82,275 @@ SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+  6 | Plantmon   | 2021-11-15    |               2 | t       |     -5.70 | 
+  7 | Squirtle   | 1993-04-02    |               3 | f       |    -12.13 | 
+  8 | Angemon    | 2005-06-12    |               1 | t       |    -45.00 | 
+  9 | Boarmon    | 2005-06-07    |               7 | t       |     20.40 | 
+ 10 | Blossom    | 1998-10-13    |               3 | t       |     17.00 | 
+ 11 | Ditto      | 2022-05-14    |               4 | t       |     22.00 | 
+(11 rows)
+*/
+
+START TRANSACTION;
+
+-- Update the species column to "digimon" for animals whose names end in "mon"
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+/*
+ id |    name    | date_of_birth | escape_attempts | neutred | weight_kg | species
+----+------------+---------------+-----------------+---------+-----------+---------
+  3 | Pikachu    | 2021-01-07    |               1 | f       |     15.04 |
+  5 | Charmander | 2020-02-08    |               0 | f       |    -11.00 |
+  7 | Squirtle   | 1993-04-02    |               3 | f       |    -12.13 |
+ 10 | Blossom    | 1998-10-13    |               3 | t       |     17.00 |
+ 11 | Ditto      | 2022-05-14    |               4 | t       |     22.00 |
+  1 | Agumon     | 2020-02-03    |               0 | t       |     10.23 | digimon
+  2 | Gabumon    | 2018-11-15    |               2 | t       |      8.00 | digimon
+  4 | Devimon    | 2017-05-12    |               5 | t       |     11.00 | digimon
+  6 | Plantmon   | 2021-11-15    |               2 | t       |     -5.70 | digimon
+  8 | Angemon    | 2005-06-12    |               1 | t       |    -45.00 | digimon
+  9 | Boarmon    | 2005-06-07    |               7 | t       |     20.40 | digimon
+(11 rows)
+*/
+
+-- Update the species column to "pokemon" for animals whose species is not set
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+/*
+ id |    name    | date_of_birth | escape_attempts | neutred | weight_kg | species
+----+------------+---------------+-----------------+---------+-----------+---------
+  1 | Agumon     | 2020-02-03    |               0 | t       |     10.23 | digimon
+  2 | Gabumon    | 2018-11-15    |               2 | t       |      8.00 | digimon
+  4 | Devimon    | 2017-05-12    |               5 | t       |     11.00 | digimon
+  6 | Plantmon   | 2021-11-15    |               2 | t       |     -5.70 | digimon
+  8 | Angemon    | 2005-06-12    |               1 | t       |    -45.00 | digimon
+  9 | Boarmon    | 2005-06-07    |               7 | t       |     20.40 | digimon
+  3 | Pikachu    | 2021-01-07    |               1 | f       |     15.04 | pokemon
+  5 | Charmander | 2020-02-08    |               0 | f       |    -11.00 | pokemon
+  7 | Squirtle   | 1993-04-02    |               3 | f       |    -12.13 | pokemon
+ 10 | Blossom    | 1998-10-13    |               3 | t       |     17.00 | pokemon
+ 11 | Ditto      | 2022-05-14    |               4 | t       |     22.00 | pokemon
+(11 rows)
+*/
```

Comment:
### Optional
- [ ] Kindly remove this commented query results, we already have the screenshot of the query results so it is not needed here. Making it difficult to understand your code